tests/e2e/webform-snapshots/webforms.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { expect, Page, test } from '@playwright/test';
+import { execSync } from 'child_process';
+import scrape from 'website-scraper';
+
+import { cmsUrl } from '../helpers/url';
+
+const baseDir = '../../packages/ui/static/public/webforms';
+
+test('Export webforms for styling', async ({ page }) => {
+  execSync(`rm -rf ${baseDir}`);
+
+  const baseUrl = cmsUrl('/en/form/styling?iframe=true&no_css=true');
+
+  await page.goto(baseUrl);
+  await savePage(page, 'idle');
+
+  await page.getByLabel('Email with a confirmation').fill('asd@asd');
+  await page.getByRole('button', { name: 'Submit' }).click();
+  await expect(
+    page.locator('[data-drupal-messages] [role="alert"]'),
+  ).toBeVisible();
+  await savePage(page, 'error');
+});
+
+async function savePage(page: Page, name: string) {
```

Comment:
Should we move `savePage` and the `ScraperPlugin` to a shared package? Maybe in a later step.

Reply:
It uses `Page` type which comes from `@playwright/test`. If we move it to a package, we can get issues with Playwright versions. So I would not.

Reply:
Or we change it to
```ts
async function savePage(url: string, content: string, name: string) {
```
and call it as
```ts
await savePage(page.url(), await page.content(), 'name');
```
but then it's not nice anymore 😅

Reply:
TBH, when thinking about testability and reusability, that makes a lot of sense 🤔 
We could use that scraper for other tasks, outside of playwright as well.

Reply:
https://www.npmjs.com/package/@amazeelabs/save-webpage